src/app/app.config.ts

```diff
@@ -23,126 +29,159 @@ export type ComponentStructure = {
 
-const components: ComponentStructure[] = [
+const components: Omit<ComponentStructure, 'id'>[] = [
   {
-    id: 1,
     label: 'Buttons',
     title: 'Button',
     description: 'Button Control',
     url: 'https://taiga-ui.dev/components/button',
     component: ButtonComponent,
   },
   {
-    id: 2,
     label: 'Buttons Close',
     title: 'Button Close',
     description: 'Close Button Control',
     url: 'https://taiga-ui.dev/components/button-close',
     component: ButtonCloseComponent,
   },
   {
-    id: 3,
     label: 'Button Group',
     title: 'Button Group',
     description: 'Grouping Buttons in one control',
     url: 'https://taiga-ui.dev/components/button-group',
     component: ButtonGroupComponent,
   },
   {
-    id: 4,
     label: 'Button Loading',
     title: 'Button Loading',
     description: 'Button with a loading wheel',
     url: 'https://taiga-ui.dev/components/button#loading',
     component: ButtonLoadingComponent,
   },
   {
-    id: 4,
     label: 'Button With DI',
     title: 'Button With DI',
     description: 'Button that user can configure using Dependency Injection',
     url: 'https://taiga-ui.dev/components/button#options-with--di',
     component: ButtonDiComponent,
   },
   {
-    id: 5,
     label: 'Button Vertical',
     title: 'Button Vertical',
     description: 'Button with Icon in a vertical design',
     url: 'https://taiga-ui.dev/components/button#vertical',
     component: ButtonVerticalComponent,
   },
   {
-    id: 6,
     label: 'Links',
     title: 'Link',
     description: 'Link Control',
     url: 'https://taiga-ui.dev/components/link',
     component: LinkComponent,
   },
   {
-    id: 7,
     label: 'Links Icons',
     title: 'Link with Icon',
     description: 'A Link with an icon',
     url: 'https://taiga-ui.dev/components/link#icons',
     component: LinkIconsComponent,
   },
   {
-    id: 8,
     label: 'Links Variants',
     title: 'Link Variants',
     description: 'Variants of how links look like',
     url: 'https://taiga-ui.dev/components/link#variants',
     component: LinkVariantsComponent,
   },
   {
-    id: 9,
     label: 'Long Text Links',
     title: 'Long Text Link',
     description: 'Setting a long text to be a link',
     url: 'https://taiga-ui.dev/components/link#long-text',
     component: LinkLongTextComponent,
   },
   {
-    id: 10,
     label: 'Icons',
     title: 'Icon',
     description: 'Icon Control',
     url: 'https://taiga-ui.dev/components/icon',
     component: IconComponent,
   },
   {
-    id: 11,
     label: 'External Icons',
     title: 'External Icons',
     description: 'Icons from external source',
     url: 'https://taiga-ui.dev/components/icon#external',
     component: IconExternalComponent,
   },
   {
-    id: 12,
     label: 'Two Colors Icons',
     title: 'Two Colors Icons',
     description: 'Icons with 2 colors',
     url: 'https://taiga-ui.dev/components/icon#two-colors',
     component: IconTwoColorsComponent,
   },
   {
-    id: 13,
```

Comment:
Why you omit the id from config?
Id prop is needed to filter by component id inside dashboard.service
<img width="265" alt="image" src="https://github.com/user-attachments/assets/ac6b33a1-5ee9-466b-8458-7cdc3121c990">

Reply:
I omit because there are an error: i found two id:4
I find a way to avoid to insert manually id.
Also i create a file so appconfig Is more readable

Reply:
Yeah, nice I did not notice that.